integration/multiarch/fixtures/create_reports.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
do we need this file if we have the helper function in the test?

Reply:
No, we don't. I've removed this file.